src/scripts/main.js

```diff
@@ -1,3 +1,66 @@
 'use strict';
 
-// write code here
+const clickArea = document.querySelector('thead');
+const dataTD = [...document.querySelectorAll('td')];
+const dataArray = [];
+const allTagsTD = dataTD.slice();
+
+function createData() {
+  const parseData = () => {
+    return allTagsTD.splice(0, 1)[0].innerText;
+  };
+
+  while (allTagsTD.length > 0) {
+    const person = {};
+
+    person.name = parseData();
+    person.position = parseData();
+    person.age = +parseData();
+    person.salary = +parseData().match(/\d/g).join('');
+    dataArray.push(person);
+  }
+}
```

Comment:
could be simplified. Just use all 'trs' and sort them, don't create new ones.

Reply:
Done it!